sdk/communication/azure-communication-identity/CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 ### Bugs Fixed
 
 ### Other Changes
+- Python 2.7 is no longer supported. Please use Python version 3.6 or later.
```

Comment:
There is no need to do a release just for dropping py 2.7 and we are not planning to release after this PR merge, it can be released with future functions, that is why I left the versions as they are. If any communication team wants to release then they could modify the version and do the release